registry/quilt_server/models.py

```diff
@@ -6,26 +6,20 @@
 
 from packaging.version import Version as PackagingVersion
 
-from sqlalchemy.dialects import mysql
 from sqlalchemy.orm import deferred
+from sqlalchemy.dialects import postgresql
 
 from . import db
 
-UTF8_BIN = 'utf8_bin'
-UTF8_GENERAL_CI = 'utf8_general_ci'
-
-def CaseSensitiveString(length):
-    return mysql.VARCHAR(length, collation=UTF8_BIN)
-
-USERNAME_TYPE = CaseSensitiveString(64)
+USERNAME_TYPE = db.String(64)
```

Comment:
fyi in postgres it's trivial to setup a real column-constraint which matches the rules for allowable usernames... CREATE FUNCTION then use in a constraint.  imho add as a TODO.

Reply:
Eh, we'll have a proper User table and integer IDs eventually